examples/catalog/test/custom_semantics_test.dart

```diff
@@ -19,11 +19,10 @@ void main() {
 
     // Verify it correctly exposes its semantics.
     // TODO(goderbauer): Use `SemanticsTester` after https://github.com/flutter/flutter/issues/12286.
-    final SemanticsNode semantics = tester
-        .renderObject(find.byType(AdjustableDropdownListTile))
-        .debugSemantics;
+    final SemanticsNode semantics = tester.renderObject(find.byType(AdjustableDropdownListTile)).debugSemantics;
```

Comment:
probably fine, i guess, though again there are cases where this could really harm readability.

Reply:
You can _force_ split with a trailing comment:

```dart
    final SemanticsNode semantics = tester //
        .renderObject(find.byType(AdjustableDropdownListTile))
        .debugSemantics;
```